source/multigrid/mg_transfer_prebuilt.cc

```diff
@@ -280,6 +280,15 @@ void MGTransferPrebuilt<VectorType>::build_matrices
                                                    mg_dof);
       dsp.reinit(0,0);
 
+      // In the end, the entries in this object will only be real valued.
+      // Nevertheless, we have to take the underlying scalar type of the
+      // vector we want to use this class with. The reason for that is
+      // that the global matrix the entries of this matrix are copied into
```

Comment:
Probably ~the global matrix~

Reply:
No, this was intended but maybe too complicated. I rephrased it a bit.